core/src/main/java/org/infinispan/container/impl/DefaultInternalDataContainer.java

```diff
@@ -43,15 +44,15 @@
  * @since 9.3
  */
 @Listener
-public class DefaultSegmentedDataContainer<K, V> extends AbstractSegmentedDataContainer<K, V> {
+public class DefaultInternalDataContainer<K, V> extends AbstractInternalDataContainer<K, V> {
```

Comment:
I think the old name actually fits better here

Reply:
Sure